libs/designsystem/src/lib/components/index.ts

```diff
@@ -63,3 +63,5 @@ export { ToggleComponent } from './toggle/toggle.component';
 
 export { ProgressCircleComponent } from './progress-circle/progress-circle.component';
 export * from './toggle-button';
+
+export * from './page-local-navigation';
```

Comment:
Always prefer explicit export, because implicit export is one of the most prominent factors for circular dependency error